pkg/updater/gcs.go

```diff
@@ -101,7 +101,7 @@ const (
 	podInfoRow = "Pod"
 )
 
-func MergeCells(cells ...Cell) Cell {
+func MergeCells(flakyFail bool, cells ...Cell) Cell {
```

Comment:
Can you comment a bit on what flakyFail does?

Reply:
#431 documents these functions (and chooses a better name for this argument)